### PR TITLE
Add fields method

### LINKED ISSUE #224: Add fields property
This issue is opened here because its fix it is needed here to address [this](https://github.com/open-telemetry/opentelemetry-python/pull/1374) issue.

### PR DESCRIPTION
**Pull request by ocelotl**
Fixes #224

# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A

# Does This PR Require a Core Repo Change?

- [X] Yes. - Link to PR: 
- [ ] No.

# Checklist:

See [contributing.md](https://github.com/open-telemetry/opentelemetry-python-contrib/blob/master/CONTRIBUTING.md) for styleguide, changelog guidelines, and more.

- [ ] Followed the style guidelines of this project
- [ ] Changelogs have been updated
- [ ] Unit tests have been added
- [ ] Documentation has been updated

### CODE REVIEW

**Review by codeboten — COMMENTED** [review]
Does this need an entry in the changelog for the aws sdk?

**Comment by ocelotl** [comment]
> Does this need an entry in the changelog for the aws sdk?

Added this changelog entry.

**Review by codeboten — APPROVED** [review]
LGTM

File: sdk-extension/opentelemetry-sdk-extension-aws/CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+- Add fields method
```
Comment (NathanielRN) [inline sdk-extension/opentelemetry-sdk-extension-aws/CHANGELOG.md:5]:
```suggestion
- Add method to return fields injected by propagator
```

**Review by NathanielRN — APPROVED** [review]
Nice! Thank you 😄 

Also, maybe a test that compares the fields returned by `fields()` and the fields set after an `inject` would be helpful?

**Comment by NathanielRN** [comment]
Here is the Core PR: https://github.com/open-telemetry/opentelemetry-python/pull/1374

**Pull request merged**